x11-packages/dosbox-x/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://dosbox-x.com/
+TERMUX_PKG_DESCRIPTION="DOSBox-X fork of the DOSBox project"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.83.22
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/joncampbell123/dosbox-x/archive/refs/tags/dosbox-x-v${TERMUX_PKG_VERSION/.3/-3}.tar.gz
+TERMUX_PKG_SHA256=
```

Comment:
This is necessary. CI will fail without it.

Reply:
oops i forgot to add back in the checksum! on it!